src/Raven.Client/Documents/Session/AsyncDocumentSession.cs

```diff
@@ -101,9 +101,6 @@ protected override Task<string> GenerateIdAsync(object entity)
         public IRevisionsSessionOperationsAsync Revisions => _revisions ?? (_revisions = new DocumentSessionRevisionsAsync(this));
         private IRevisionsSessionOperationsAsync _revisions;
 
-        public ICountersSessionOperationsAsync Counters => _counters ?? (_counters = new DocumentSessionCountersAsync(this));
```

Comment:
We do not want to have counters in .Advanced? It is a bit inconsistent with the rest of the session API.

Reply:
Yes, because it got really verbose and awkward when we started using it.
We also want to make this a 1st class item.
We'll probably also add `AttachmentsFor` to make this more discoverable